### PR TITLE
vosviewer: add version 1.6.18

### LINKED ISSUE #8878: [Request] VOSviewer
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** VOSviewer

**Description:** VOSviewer is a software tool for constructing and visualizing bibliometric networks. These networks may for instance include journals, researchers, or individual publications, and they can be constructed based on citation, bibliographic coupling, co-citation, or co-authorship relations. VOSviewer also offers text mining functionality that can be used to construct and visualize co-occurrence networks of important terms extracted from a body of scientific literature.

**Homepage:** [https://www.vosviewer.com/](https://www.vosviewer.com/)

**Download link(s):** [https://www.vosviewer.com/download](https://www.vosviewer.com/download)

**Some indication of popularity/repute:** 100s of applied scientific publications: [https://www.vosviewer.com/publications#applied-publications](https://www.vosviewer.com/publications#applied-publications)

### PR DESCRIPTION
**Pull request by Lutra-Fs**
Closes #8878

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Lutra-Fs** [comment]
Hi I have created a manifest for VOSviewer but it depends on jre8. I think all java related software has been moved to java bucket. how to solve the dependecy issue?

**Comment by Lutra-Fs** [comment]
I have tested Eclipse Temurin JRE with Hotspot 8 but no luck. Seems like it depends on Oracle Java 8 to start

**Comment by Lutra-Fs** [comment]
Update: I tried ```oraclejre8``` in ```java``` bucket but it does not work

**Comment by Lutra-Fs** [comment]
seems like a full jre8 installation via https://www.java.com/en/download/ is required

**Comment by issaclin32** [comment]
> Update: I tried `oraclejre8` in `java` bucket but it does not work

Probably related to this issue: https://github.com/ScoopInstaller/Java/issues/119

**Comment by Lutra-Fs** [comment]
> I have tested Eclipse Temurin JRE with Hotspot 8 but no luck. Seems like it depends on Oracle Java 8 to start

I installed openjre via winget so maybe Java is required for this program

**Comment by Lutra-Fs** [comment]
> > Update: I tried `oraclejre8` in `java` bucket but it does not work
> 
> > seems like a full jre8 installation via [java.com/en/download](https://www.java.com/en/download/) is required
> 
> Probably related to this issue: [ScoopInstaller/Java#119](https://github.com/ScoopInstaller/Java/issues/119)

Agreed. Just found that termuin jre installed via winget will not add Oracle Java related registry keys. After manually changing it now worked as expected

**Comment by Lutra-Fs** [comment]
So maybe add a notes to [ScoopInstaller/Java#119](https://github.com/ScoopInstaller/Java/issues/119)? Or waiting for java bucket to add support? but as I have seen that there is no progress since 2020.

**Comment by issaclin32** [comment]
> So maybe add a notes to [ScoopInstaller/Java#119](https://github.com/ScoopInstaller/Java/issues/119)? Or waiting for java bucket to add support? but as I have seen that there is no progress since 2020.

I think adding `notes` should be fine.

I am not an expert on *Java*. I am not really sure why *Java* bucket's maintainer did not solve that, maybe they have some other concerns.

**Comment by Lutra-Fs** [comment]
> > So maybe add a notes to [ScoopInstaller/Java#119](https://github.com/ScoopInstaller/Java/issues/119)? Or waiting for java bucket to add support? but as I have seen that there is no progress since 2020.
> 
> I think adding `notes` should be fine.
> 
> I am not an expert on _Java_. I am not really sure why _Java_ bucket's maintainer did not solve that, maybe they have some other concerns.

Added

File: bucket/vosviewer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.6.18",
+    "description": "VOSviewer is a software tool for constructing and visualizing bibliometric networks.",
+    "homepage": "https://www.vosviewer.com/",
+    "license": "Proprietary",
+    "notes": " VOSviewer depends on registry keys to detect your java installation. If you have JRE 8 installed but meet error on running VOSviewer, please refers to https://github.com/ScoopInstaller/Java/issues/119 for more information.",
```
Comment (issaclin32) [inline bucket/vosviewer.json:6]:
```suggestion
    "notes": [
        "VOSviewer depends on registry keys to detect your java installation. ",
        "If you have JRE 8 installed but meet error on running VOSviewer",
        "use the installer from https://www.oracle.com/java/",
        "or, refer to https://github.com/ScoopInstaller/Java/issues/119 for solutions"
    ],
```

Comment (Lutra-Fs) [reply bucket/vosviewer.json:6]: Should I remove suggest?

**Comment by Lutra-Fs** [comment]
Should I remove her suggestion in manifest?

**Comment by issaclin32** [comment]
> Should I remove jre suggestion in manifest?

Yes. Please remove it.

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**